src/Util/PHP/Template/TestCaseMethod.tpl.dist

```diff
@@ -47,7 +47,12 @@ function __phpunit_run_isolated_test()
     $result->beStrictAboutTodoAnnotatedTests({isStrictAboutTodoAnnotatedTests});
     $result->beStrictAboutResourceUsageDuringSmallTests({isStrictAboutResourceUsageDuringSmallTests});
 
-    $test = new {className}('{methodName}', unserialize('{data}'), '{dataName}');
```

Comment:
Would it make sense to use a new, separate template here? Maybe `TestCaseClass.tpl.dist`.